README.md

```diff
@@ -1,18 +1,42 @@
-# Minimal Extension
+# Experimental browser extension
 
-> This is minimal extension to explore communication between a browser page and
-> an extension popup
+Securely connect to Vega dapps and sign transactions from your browser.
+
+This is an experimental release for test purposes only and is in active development. You should regard all funds transferred to keys that are managed by the extension as lost.
+
+Installing the extension injects a Vega API for dapps and websites to interact with keys and send transactions to the Vega network. Connecting to the wallet and sending any transaction require explicit user approval.
+
+**Understand the risks before you start**
+
+- You retain custody of your assets
+- You are responsible for the security of your assets
+- You should never share your private keys
+- You may encounter bugs, loss of functionality or loss of assets using the software
+- No party accepts any liability for any losses whatsoever relating to its use
+
+By using the Vega Wallet, you acknowledge that you have read and understood the [Vega Wallet User Data Policy ↗](https://vega.xyz/vega-wallet-user-data-policy/)
 
 ## Installation
 
-**NOTE** The extension is in active development and may break at any time.
-It will not allow you to backup the recovery phrase, and you should regard
-all funds transferred to keys that are managed to the extension as lost.
+Only one extension (Fairground or Mainnet) should be installed at a time to avoid version conflict. Updates may wipe all data, and keys are not recoverable.
```

Comment:
```suggestion
Only one extension should be installed at a time to avoid version conflict. Updates may wipe all data, and keys are not recoverable. If you are communicating with multiple networks, consider using multiple browser profiles.
```